infor-design/enterprise
# 3423 - Fix double change events in datepicker

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Change fired twice in datepicker. This fixes..

**Related github/jira issue (required)**:
Fixes #3423 

**Steps necessary to review your pull request (required)**:
**Test One**
- go to http://localhost:4000/components/datepicker/test-change-event.html
- click in first field
- then hit tab
- no toast should show (no event should fire)

**Test Two**
- go to http://localhost:4000/components/datepicker/test-change-event.html
- click in second field
- then hit tab
- no toast should show (no event should fire)

**Test Three**
- click in first field
- type 5/5/2020
- then hit tab
- one toast should show (event should fire)
- reload
- click in first field
- select a value in the picker
- focus the input
- then hit tab
- one toast should show (event fired on selection not tab)

**Test Four**
- click in second field
- type over the value and make it 5/5/2020
- then hit tab
- one toast should show (event should fire)

Essentially only one toast should show per change.

**Included in this Pull Request**:
